ruleset/testing/tests/nextcloud.ini

```diff
@@ -1,48 +1,59 @@
+;  Copyright (C) 2015-2021, Wazuh Inc.
+;
+;  Tests for products:
+;    NextCloud.
+
```

Comment:
Some rules test are missing (88204 to 88209 )

Reply:
Can't generate logs for nextcloud firewall